Duplication of Entities type within entity.ts

This appears to only affect TypeScript 3.7+, as per https://devblogs.microsoft.com/typescript/announcing-typescript-3-7-rc/#local-and-imported-type-declarations-now-conflict

The `Entities` type is imported: https://github.com/googleapis/nodejs-datastore/blob/master/src/entity.ts#L22
And a separate `Entities` type is defined and exported: https://github.com/googleapis/nodejs-datastore/blob/master/src/entity.ts#L1288

TypeScript 3.7.1-rc reports
```
node_modules/@google-cloud/datastore/build/src/entity.d.ts:19:10 - error TS2440: Import declaration conflicts with local declaration of 'Entities'.

19 import { Entities } from './request';
```

--- a/src/entity.ts
+++ b/src/entity.ts
@@ -19,7 +19,6 @@ import * as extend from 'extend';
 import * as is from 'is';
 import {Query, QueryProto} from './query';
 import {PathType} from '.';
-import {Entities} from './request';
 import * as Protobuf from 'protobufjs';
 import * as path from 'path';
 import * as appengine from '../proto/app_engine_key';

--- a/src/request.ts
+++ b/src/request.ts
@@ -38,6 +38,7 @@ import {
   KeyProto,
   ValueProto,
   ResponseResult,
+  Entities,
 } from './entity';
 import {
   Query,
@@ -1376,7 +1377,6 @@ export interface BooleanObject {
 export interface ConsistencyProtoCode {
   [key: string]: number;
 }
-export type Entities = Entity | Entity[];
 export interface EntityProtoReduceAccumulator {
   [key: string]: ValueProto;
 }

